.github/workflows/deploy_staging.yml

```diff
@@ -176,6 +176,10 @@ jobs:
 
             - name: Deploy backend staging CloudFormation stack
               run: pnpm --filter "antalmanac-cdk" backend-staging deploy
+              env:
+                  GOOGLE_CLIENT_ID: ${{ secrets.GOOGLE_CLIENT_ID }}
+                  GOOGLE_CLIENT_SECRET: ${{ secrets.GOOGLE_CLIENT_SECRET }}
+                  GOOGLE_REDIRECT_URI: ${{ needs.get_staging_configuration.outputs.backend_environment_url }}
```

Comment:
Needs correcting to match endpoint

Reply:
`${{ needs.get_staging_configuration.outputs.backend_environment_url }}` resolves to the backend URL of this staging instance. If this should be the frontend URL, change it to `${{ needs.get_staging_configuration.outputs.frontend_environment_url }}`. You can add text after it to specify the endpoint. (Note that spaces are significant.)